sql/src/test/resources/essetup/data/copy/test_copy_from.csv

```diff
@@ -0,0 +1,4 @@
+id,quote
+1,"Don't pañic."
```

Comment:
is it intentional that the "n" has a tilde?

not docs, but I noticed this whilst scrolling. same thing in the following two test files

Reply:
yes, it's intended. the `json` test has this also: https://github.com/crate/crate/blob/master/sql/src/test/resources/essetup/data/copy/test_copy_from.json
I believe it's to test the encoding

Reply:
aaah got you!